### DISCUSSION TITLE
Stickers without censor

### DISCUSSION THREAD

**Discussion by nisaeggu** [silveredgold/beta-protection#22] [Ideas]
The "sticker" and "caption" option is currently always censoring the area beneath the sticker. If there would be an option for stickers without censoring that would be great because then you could add anything on top, for example borders or sparkle effects using .png's. 

Maybe something like
- Sticker (blur off)
- Sticker (blur below)

**Comment by silveredgold** [comment]
Assuming you're using Beta Censoring as your backend, you can't disable the censoring under the sticker since the server can't guarantee that stickers will always cover the censoring area. You can however set it to use pixelation instead of blurring using [this configuration option](https://github.com/silveredgold/beta-censoring/issues/18#issuecomment-1379681121).
